account_payment_order/models/account_move_line.py

```diff
@@ -85,6 +85,7 @@ def _prepare_payment_line_vals(self, payment_order):
             "communication_type": communication_type,
             "currency_id": currency_id,
             "amount_currency": amount_currency,
+            "date": False,
```

Comment:
Why forcing this to False?

Reply:
If I'm not mistaken, because it was possible that I put today's date

Reply:
I see that is a related field: https://github.com/OCA/bank-payment/blob/00031b13ff311faed526df86c5fdd132d922b294/account_payment_order/models/bank_payment_line.py#L65, so all of this seems weird.

Reply:
But the question is not the default or not, but the related part.

Reply:
In the original version:
# date is set when the user confirms the payment order
I think it is better to set the field False, just in case.

Reply:
But if this field is related, setting to anything is trickling the system.

Reply:
I think the field you indicated before is not correct. The correct field is this:
https://github.com/OCA/bank-payment/blob/00031b13ff311faed526df86c5fdd132d922b294/account_payment_order/models/account_payment_line.py#L62

Reply:
Oh, right. Sorry for the confusion.